build-windows.rb

```diff
@@ -170,10 +173,15 @@ def display_reminders()
 ################################################################################
 #                          Do The Build                                        #
 ################################################################################
+if ENV['USER'] != "root" then
+   puts "Run this script as a Superuser, or the buildscript wont work!"
+   abort
+end   
```

Comment:
The buildscripts should *not* be run as root. Any cases where root permissions are currently needed are obtained via sudo.

Reply:
This was maybe worded wrong. I meant to run the script as sudo, but no problem i will change that.